content/en/docs/tasks/administer-cluster/namespaces.md

```diff
@@ -78,7 +78,7 @@ A namespace can be in one of two phases:
    * `Active` the namespace is in use
    * `Terminating` the namespace is being deleted, and can not be used for new objects
 
-See the [design doc](https://git.k8s.io/community/contributors/design-proposals/architecture/namespaces.md#phases) for more details.
+See the [design doc](https://kubernetes.io/docs/reference/kubernetes-api/cluster-resources/namespace-v1/) for more details.
```

Comment:
```suggestion
For more details, see [Namespace](/docs/reference/kubernetes-api/cluster-resources/namespace-v1/)
in the API reference.
```

Reply:
To make this work: we should amend [k/kubernetes](https://github.com/kubernetes/kubernetes) to mention these details in the doc comments for the Namespace API code:

> A Namespace is in the Active phase if it does not have a ObjectMeta.DeletionTimestamp.
>
> A Namespace is in the Terminating phase if it has a ObjectMeta.DeletionTimestamp.

If accepting this suggestion, we should then /hold this PR until we do that.

Reply:
ok, @sftim I got your point, I commit your proposed changes, let's just hold till that Namespace API code being amended. Thanks for your help.

Reply:
Done with the requested changes